src/odbc/src/statement.cpp

```diff
@@ -551,13 +575,25 @@ SqlResult::Type Statement::InternalGetAttribute(int attr, void* buf, SQLINTEGER,
       break;
     }
 
+    case SQL_ROWSET_SIZE: {
+      SqlUlen* val = reinterpret_cast<SqlUlen*>(buf);
+
+      *val = rowsetSize;
```

Comment:
It wasn't checked anywhere but doesn't seem to cause problems. There is even a test for it:
```c++
  SQLGetStmtAttr(stmt, SQL_ATTR_ROW_ARRAY_SIZE, 0, 0, 0);
```

I have added null checks just in case for all `val` values in `InternalGetAttribute`.